prost-types/src/lib.rs

```diff
@@ -395,6 +395,16 @@ impl std::hash::Hash for Timestamp {
     }
 }
 
+#[cfg(feature = "std")]
+impl Ord for Timestamp {
```

Comment:
I believe you could just `#[derive(PartialOrd, Ord)]` on `struct Timestamp`. At least your unit test succeeds.

Reply:
I noticed a potential issue in both the comparison logic when dealing with timestamps. For instance, when comparing:
`Timestamp {
seconds: 1,
nanos: 0 }`

to

`Timestamp {
seconds: 0,
nanos: 1_000_000_000
}`

The current implementation considers the first timestamp as greater than the second one. However, it's crucial to recognize that the second timestamp actually represents a same duration.

Reply:
There is a normalize function in Timestamp. I believe that does what you are looking for. https://docs.rs/prost-types/latest/prost_types/struct.Timestamp.html#method.normalize